src/Parser/Rogue/Assassination/Modules/Features/Abilities.js

```diff
@@ -0,0 +1,89 @@
+import SPELLS from 'common/SPELLS';
+import CoreAbilities from 'Parser/Core/Modules/Abilities';
+
+class Abilities extends CoreAbilities {
+  spellbook() {
+    const combatant = this.combatants.selected;
+    return [
+      {
+        spell: SPELLS.VANISH,
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        cooldown: 120,
+      },
+      {
+        spell: SPELLS.VENDETTA, // TODO: Reduced by Convergence of Fates
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        cooldown: 180,
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 1.0,
+          extraSuggestion: 'In most cases should be used on cooldown',
+        },
+      },
+      {
+        spell: SPELLS.TOXIC_BLADE_TALENT, 
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        cooldown: 25,
```

Comment:
What was the reason for putting a 25 second CD as a cooldown?  You've got 20 second abilities listed as rotational so I'm just curious

Reply:
Not sure what the distinction should be TBH. I guess its more related to how I personally treat the spell. Is there any guidelines to what should be considered cooldown for the purpose of wowanalyzer?

Reply:
There is no official distinction, it's pretty subjective. You can keep it or change it at your discretion.